engine/src/test/scala/cromwell/RetryableCallsSpec.scala

```diff
@@ -27,25 +25,27 @@ class RetryableCallsSpec extends CromwellTestkitSpec with Mockito {
     }
   }
 
+  val customizedLocalBackendOptions =
+    """
+      |{
+      |  "backend": "customizedLocalBackend"
+      |}"""".stripMargin
```

Comment:
I just tried running a workflow with a trailing `"` in workflow options JSON file and it worked.  Interesting.